src/schema/users.ts

```diff
@@ -669,6 +669,7 @@ export const typeDefs = /* GraphQL */ `
     total: Int
     current: Int
     lastViewAt: DateTime
+    lastViewAtTz: DateTime
```

Comment:
Why do we need the both timezoned and utc?

Reply:
Old one is just for backwards support. The one with timezone is for this PR: https://github.com/dailydotdev/apps/pull/3832